frontend/src/main/web/src/components/ButtonLink/readme.md

```diff
@@ -0,0 +1,17 @@
+A button that looks like a link. Should only be used for page interaction,
+if you need to link to a url, use the `<Link />` component.
+
+## Default
+
+    <ButtonLink>Button Link</ButtonLink>
+
+## Types
+
+    const types = require('../../constants/styles').types;
+    <div>
+      {types.map((type, i) =>
+        <ButtonLink key={i} type={type} atomic={{m: 'Mend(rh)', tt: 'Tt(c)'}}>
+          {type}
+        </ButtonLink>
+      )}
+    </div>
```

Comment:
Is this block generating a style guide entry that shows each of the possible types that can be used?